lib/galaxy/objectstore/__init__.py

```diff
@@ -483,7 +491,13 @@ def file_ready(self, obj, **kwargs):
 
     def create(self, obj, **kwargs):
         """Create a backing file in a random backend."""
-        random.choice(list(self.backends.values())).create(obj, **kwargs)
+        self.set_object_store_id(obj)
+        return self._call_method('create', obj, False, False, **kwargs)
```

Comment:
It seems to me that this is not actually calling the `create` method on any backend, when the object does not exist in any backend yet.